opts/opts.go

```diff
@@ -130,6 +172,15 @@ func (opts *ListOpts) Len() int {
 type ValidatorFctType func(val string) (string, error)
 type ValidatorFctListType func(val string) ([]string, error)
 
+func ValidateLogOpts(val string) (string, error) {
+	allowedKeys := map[string]string{}
+	vals := strings.Split(val, "=")
```

Comment:
@LK4D4 This cannot be `strings.Contains`. `strings.Contains` will validate `some-string=allowed-string`, whereas we only want `allowed-string=some-string` to be valid.